src/site/content/en/blog/optimize-cls/index.md

```diff
@@ -180,138 +259,46 @@ Chrome, Firefox and Safari now support setting `width` and `height` on the `sour
 
-#### Statically reserve space for the ad slot
+#### Statically reserve space for late-loading content
 
-Statically style slot DOM elements with the same sizes passed to your tag library. This can help ensure the library doesn't introduce layout shifts when it loads. If you don't do this, the library may change the size of the slot element after page layout.
+If placing late-loading content in the content flow, ensure shifts are eliminated by reserving the space for them in the initial layout.
 
-This can be as simple as adding a `min-height` styling to reserve space or, for responsive ad sizings using the new [`aspect-ratio`](/aspect-ratio/) CSS property in a similar manner to the way browsers automatically use this for images with dimensions provided.
+This can be as simple as adding a `min-height` styling to reserve space or, for responsive content such as ads, using the new [`aspect-ratio`](/aspect-ratio/) CSS property in a similar manner to the way browsers automatically use this for images with dimensions provided.
 
 <figure>
-  {% Img src="image/W3z1f5ZkBJSgL1V1IfloTIctbIF3/ThcGvVp0RiiABpmnWz7u.svg", alt="Three mobile devices with jjst text content in the first device, this is shifted down in the second device, and reserving space with a placehodler as shown in the third device prevents the shift", width="1180", height="600" %}
+  {% Img src="image/W3z1f5ZkBJSgL1V1IfloTIctbIF3/ThcGvVp0RiiABpmnWz7u.svg", alt="Three mobile devices with just text content in the first device, this is shifted down in the second device, and reserving space with a placeholder as shown in the third device prevents the shift", width="1180", height="600" %}
   <figcaption>
     Reserving space for ads can prevent layout shifts
   </figcaption>
 </figure>
 
-Also consider the sizes of smaller ad serves. If a smaller ad is served, a publisher can style the (larger) container to avoid layout shifts. The downside to this approach is that it will increase the amount of blank space, so keep in mind the trade-off here.
+You may need to account for subtle differences in ad/placeholder sizes between different form factors using media queries.
 
-Alternatively, set the initial size to the smallest ad size, and accept some level of shift for larger ads. This will not fully eliminate CLS, but will hopefully reduce the impact of it to a more managable level.
+For content that may not have a fixed height—like ads you may not be able to reserve the exact correct amount. If a smaller ad is served, a publisher can style the (larger) container to avoid layout shifts. Or choose the most likely size for the ad slot based on historical data. The downside to this approach is that it will increase the amount of blank space, so keep in mind the trade-off here.
```

Comment:
```suggestion
For content that may not have a fixed height—like ads—you may not be able to reserve the exact amount of space needed to eliminate the layout shift entirely. If a smaller ad is served, a publisher can style the (larger) container to avoid layout shifts, or choose the most likely size for the ad slot based on historical data. The downside to this approach is that it will increase the amount of blank space, so keep in mind the trade-off here.
```